app/assets/javascripts/components/sampleUpload.jsx

```diff
@@ -217,7 +217,7 @@ class SampleUpload extends React.Component {
 
   filePathValid(str) {
     var regexPrefix = /^s3:\/\//;
-    var regexSuffix = /(\A[^\s\/]+\.fastq.gz)/igm;
+    var regexSuffix = /([^\s\/]+\.fastq.gz)/igm;
```

Comment:
Also the bug Rebecca reported is really from the preload s3 path, you only do the check when the preload s3 path is not empty.

Reply:
hey @yunfangjuan, are u referring to the preload result path, cos that field is optional